app/assets/stylesheets/navbar.scss

```diff
@@ -175,7 +175,7 @@
   border: 1px solid $primary-green;
   cursor: pointer;
   font-size: 18px;
-  height: 40px;
+  height: 38px;
```

Comment:
Instead of this, a better approach maybe
1. Remove height
2. Remove margins
3. Add a `margin: 5px`

Reply:
ok will try

Reply:
> Instead of this, a better approach maybe
> 
> 1. Remove height
> 2. Remove margins
> 3. Add a `margin: 5px`

done!!